pokemongo_bot/cell_workers/update_title_stats.py

```diff
@@ -89,7 +101,12 @@ def work(self):
         # If title is empty, it couldn't be generated.
         if not title:
             return WorkerResult.SUCCESS
-        self._update_title(title, _platform)
+
+        if self.terminal_title is True:
```

Comment:
This can be refactored to `if self.terminal_title`, you don't need to explicitly check if it's `True`. Same to the other `if` right below this.